apps/docs/pages/guides/database/column-encryption.mdx

```diff
@@ -9,279 +9,7 @@ export const meta = {
-
-Duplicate ciphertexts cannot be used to “attack the key”, it can only reveal the duplication. However, duplication is still information. In our examples so far, an attacker might be able to use this information to determine that two accounts share the same secret. While not technically breaking the encryption, this still leaks information to an attacker.
-
-```sql
-create table secrets (
-  id bigserial primary key,
-  secret text,
-  key_id uuid not null default 'e348034b-3f07-4878-aad6-000511d12826'::uuid,
-  nonce bytea default pgsodium.crypto_aead_det_noncegen()
-);
-
-security label for pgsodium
-  on column secrets.secret
-  is 'ENCRYPT WITH KEY COLUMN key_id NONCE nonce';
-```
-
-This is the most secure form of TCE - there is a unique key ID and a unique nonce per row.
-
-### One Key ID per Row with Associated Data
-
-The encryption that is used for TCE is one of a family of functions provided by **libsodium** to do Authenticated Encryption with Associated Data or [AEAD Encryption](https://en.wikipedia.org/wiki/Authenticated_encryption). The “associated” data is plaintext (unencrypted) information that is mixed into the authentication signature of the encrypted data, such that when you authenticate the data, you also know that the associated data is authentic.
-
-AEAD is helpful because often you have metadata associated with a secret, which isn't confidential but must not be forged.
-
-In our secret example, we might associate a "`secret`" with an "`account_id`". But what if a malicious actor wanted to use someone else's secret on their own account? If someone could forge the `account_id` data column, swapping an `account_id` with their own `account_id`, then you could be tricked into using the wrong secret. By “associating” the `account_id` with the `secret`, it cannot be forged without throwing an error.
-
-Like above, this is done simply by extending the security label with the associated data column:
-
-```sql
-create table secrets (
-  id bigserial primary key,
-  secret text,
-  account_id integer,
-  key_id uuid not null default 'e348034b-3f07-4878-aad6-000511d12826'::uuid,
-  nonce bytea default pgsodium.crypto_aead_det_noncegen()
-);
-
-security label for pgsodium
-  on column secrets.secret
-  is 'ENCRYPT WITH KEY COLUMN key_id ASSOCIATED (account_id) NONCE nonce';
-```
-
-The new label indicates which column is to be associated with the secret, and that's it! Your `account_id` and secret are now protected under the same authentication signature as the secret itself.
-
-## Resources
-
-- [Supabase Vault](/docs/guides/database/vault)
-- Read more about Supabase Vault in the [blog post](https://supabase.com/blog/vault-now-in-beta)
-- [Supabase Vault on GitHub](https://github.com/supabase/vault)
+These docs have moved. See [pg_sodium](/docs/guides/database/extensions/pgsodium)
```

Comment:
Is there a reason not to use an automated redirect through next config?

Reply:
didn't know how. fixed